guides/doc-Upgrading_and_Updating/topics/upgrading_discovery_satellite.adoc

```diff
@@ -2,15 +2,24 @@
 
 = Upgrading Discovery on {ProjectServer}
 
-. Update the Discovery template in the {ProjectWebUI}:
+. Update the Discovery template in the {ProjectWebUI}:.
 .. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates*.
 .. On the `PXELinux global default` line, click *Clone*.
 .. Enter a new name for the template in the *Name* field, for example `ACME PXE global default`.
 .. In the template editor field, change the line `ONTIMEOUT local` to `ONTIMEOUT discovery` and click *Submit*.
-.. In the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+.. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Provisioning*.
```

Comment:
We generally don't use two `>`. Instead, you can add an extra step starting with ".. On the *Provisioning* tab, select/set/click ...".